.gitignore

```diff
@@ -0,0 +1,7 @@
+venv
```

Comment:
not sure if there's some clever git thing i'm missing but there didn't seem to be a gitignore